src/components/NetworkModal/index.tsx

```diff
@@ -0,0 +1,144 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { observer } from 'mobx-react';
+
+import { Modal } from '../Modal';
+import Option from './Option';
+import { getChains } from 'provider/connectors';
+import { useContext } from '../../contexts';
+import { useRpcUrls } from 'provider/providerHooks';
+import { InjectedConnector } from '@web3-react/injected-connector';

```

Comment:
Beautiful work! 

should we keep imports order as:

Libs (react, mobx, etc)
Components 
Utils 
Styles

?
seems a nice practice consistent along many es6 projects.  is a sugar touch, just curious...